url_shortener_proj/resources/views/details.blade.php

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
+@include('parts._head')
+<body>
+<div class="col-md-6 col-md-offset-3" style="text-align: center">
+    <br>
+    <div class="col-md-6-col-md-offset-3">
+        <strong>Original Url:</strong>
+        <a href="{{$url->original}}">{{$url->original}}</a>
+    </div>
+    <br>
+    <div class="col-md-6-col-md-offset-3">
+        <strong>Generated Short Url:</strong>
+        <a href="{{route('urls.all', $url)}}" target="_blank">{{$url->short}}</a>
```

Comment:
What is the actual short URL? What can I copy and send to others once I generate the shorter version?